index.d.ts

```diff
@@ -662,7 +677,7 @@ declare namespace Eris {
       event: "channelRecipientAdd" | "channelRecipientRemove",
       listener: (channel: GroupChannel, user: User) => void
     ): T;
-    (event: "channelUpdate", listener: (channel: AnyChannel, oldChannel: OldChannel) => void): T;
+    (event: "channelUpdate", listener: (channel: AnyGuildChannel, oldChannel: OldChannel) => void): T;
```

Comment:
Although I agree, with this, it would basically be a breaking change for any GroupChannel bots.

Reply:
@Skillz4Killz to make that work, OldChannel would have to be changed a lot, it is completely wrong for Group DM bots.

Reply:
Ohh

Reply:
I agree I don't think it is worth it to support GroupChannels so I agree this should be changed. I just think when merging this should be labelled as breaking.